test/unit/player.test.js

```diff
@@ -824,3 +824,19 @@ expect(3);
   player.language('en-GB');
   strictEqual(player.localize('Good'), 'Brilliant', 'Ignored case');
 });
+
+test('should return correct values for canPlayType', function(){
+  var tag = TestHelpers.makeTag();
+  var container = document.createElement('div');
+  var fixture = document.getElementById('qunit-fixture');
+
+  container.appendChild(tag);
+  fixture.appendChild(container);
+
+  var player = new Player(tag, { techOrder: ['techFaker'] });
```

Comment:
if you use `TestHelpers.makePlayer` it'll add the tag to the fixture for you and you won't have to do lines 830-834.